### ISSUE TITLE
Exception이 발생하네요..

### ISSUE THREAD

**Issue by yjssin** [adfit/adfit-android-sdk#12]
Fatal Exception: android.util.AndroidRuntimeException
android.webkit.WebViewFactory$MissingWebViewPackageException: Failed to load WebView provider: No WebView installed
android.webkit.WebViewFactory.getProviderClass (WebViewFactory.java:438)
--
  | android.webkit.WebViewFactory.getProvider (WebViewFactory.java:211)
  | android.webkit.WebView.getFactory (WebView.java:2467)
  | android.webkit.WebView.ensureProviderCreated (WebView.java:2462)
  | android.webkit.WebView.setOverScrollMode (WebView.java:2527)
  | android.view.View. (View.java:4830)
  | android.view.View. (View.java:5016)
  | android.view.ViewGroup. (ViewGroup.java:597)
  | android.widget.AbsoluteLayout. (AbsoluteLayout.java:55)
  | android.webkit.WebView. (WebView.java:636)
  | android.webkit.WebView. (WebView.java:581)
  | android.webkit.WebView. (WebView.java:564)
  | com.kakao.adfit.ads.g. (SourceFile:100)
  | com.kakao.adfit.ads.g. (SourceFile:96)
  | com.kakao.adfit.ads.ba.BannerAdView.a (SourceFile:463)
  | com.kakao.adfit.ads.ba.BannerAdView.a (SourceFile:513)
  | com.kakao.adfit.ads.ba.BannerAdView.b (SourceFile:56)
  | com.kakao.adfit.ads.ba.BannerAdView$8.onResponse (SourceFile:1003)
  | com.kakao.adfit.ads.ba.BannerAdView$8.onResponse (SourceFile:990)
  | com.kakao.adfit.common.b.b.a (SourceFile:31)
  | com.kakao.adfit.common.b.b.a (SourceFile:12)
  | com.kakao.adfit.common.c.f$a.run (SourceFile:99)
  | android.os.Handler.handleCallback (Handler.java:789)
  | android.os.Handler.dispatchMessage (Handler.java:98)
  | android.os.Looper.loop (Looper.java:164)
  | android.app.ActivityThread.main (ActivityThread.java:6944)
  | java.lang.reflect.Method.invoke (Method.java)
  | com.android.internal.os.Zygote$MethodAndArgsCaller.run (Zygote.java:327)
  | com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1374)

Webview쪽에 문제가 있는듯한데, adfit적용전에는 보고된적 없는 crash입니다. 확인 부탁드립니다.
보고된 단말기는 삼성A8(오레오), 갤럭시On(누가) 단말기였습니다.

**Comment by chanyhan** [comment]
@yjssin 혹시 사용자 문의가 들어오면, 구글스토어에서 Android 시스템 WebView 업데이트 제거후 다시 업데이트로 안내 부탁드립니다.

**Comment by yjssin** [comment]
네 WebView쪽 문제는 사용자 단말자체 문제라 업데이트해달라고 안내하면 될거 같습니다. 허나 문제가 있다고 해서 죽으면 안되서 예외처리만 좀 부탁드립니다.

**Comment by yjssin** [comment]
대략 언제 정도 패치 예정이신가요? 조만간 저희앱도 패치 예정인데 adfit도 업데이트 적용해서 패치할지 말지 결정해야 해서요
